gtsfm/frontend/correspondence_generator/keypoint_aggregator/keypoint_aggregator_dedup.py

```diff
@@ -15,11 +15,16 @@
 
 
 class KeypointAggregatorDedup(KeypointAggregatorBase):
-    """Keypoint aggregator with de-duplication."""
+    """Keypoint aggregator with de-duplication of keypoints within each image."""
 
-    def __init__(self) -> None:
-        """Initialize global variables"""
+    def __init__(self, nms_radius: float = 3) -> None:
+        """Initialize global variables.
+
+        Args:
+            nms_radius: Radius (in pixels) to use when merging detections within the same view.
```

Comment:
Non-maximum suppression makes it seem like tracks are being suppressed instead of merged. Maybe changed to `merge_radius`?

Reply:
Updated to `nms_merge_radius` -- hopefully that captures both.

Reply:
Non-maximum suppression merge radius seems like a bit of an oxymoron...